GNUmakefile

```diff
@@ -78,4 +78,15 @@ license-check:
 tools:
 	go generate -tags tools tools/tools.go
 
-.PHONY: build test testall testacc cassettes vet fmt fmtcheck errcheck test-compile tools get-test-deps license-check
+docs: tools
+	tfplugindocs
+	@#This resource's documentation isn't currently generated
+	@rm -rf docs/resources/slo_correction.md
+	@# Timeboard + Screenbaord is deprecated
+	@# There is an issue with the security_monitoring schema that requires the docs to be updated by hand if needed
+	@git checkout HEAD -- docs/resources/timeboard.md docs/resources/screenboard.md docs/data-sources/security_monitoring_rules.md docs/resources/security_monitoring_default_rule.md docs/resources/security_monitoring_rule.md
```

Comment:
Since we are resetting these files to HEAD, would it makes sense to check if these files have uncommitted changes prior to running the tfplugindocs command?

Reply:
Would the goal there be to catch the case of someone manually modifying a file in the `docs` folder? 

My thought was that `tfplugindocs` would wipe out any changes since the last commit. And if it were a manual change, it'd be in the `git diff`. Not sure if I missed a piece there 🙂